logging/logmanager/plugin_impl_log_manager.go

```diff
@@ -120,7 +128,12 @@ func (lm *Plugin) Init() error {
 					logCfgEntry.Name, err)
 			}
 		}
-
+		lm.Log.Info("configuring log hooks")
```

Comment:
This can be just debug level or you could put all this into if len(lm.Conf.Hooks) > 0